clang/tools/clang-format/git-clang-format

```diff
@@ -260,15 +265,13 @@ def main():
                 "Ignoring the following files (wrong extension, symlink, or "
                 "ignored by clang-format):"
             )
-            for filename in ignored_files:
-                print("    %s" % filename)
+            print_filenames(ignored_files, opts.print0)
```

Comment:
Is verbose and null a combination that should be supported?

Reply:
You have a good point, should they be mutually exclusive? I.e. if both are specified an error message is printed. I don't think `--verbose` is meaningful because `--null` does not have extra information to print.

Reply:
@HazardyKnusperkeks Do you have an input on this? Right now it's just ignoring `--verbose`.